values.yaml

```diff
@@ -89,9 +95,9 @@ graphdb:
     clusterCreationTimeout: 60
     # -- Cluster configuration parameters:
     # The minimum wait time in milliseconds for a heartbeat from a leader.
-    electionMinTimeout: 7000
+    electionMinTimeout: 8000
     # The variable portion of each waiting period in milliseconds for a heartbeat.
-    electionRangeTimeout: 5000
+    electionRangeTimeout: 6000
```

Comment:
why do we change those?

Reply:
The GraphDB defaults seem to have changed

Reply:
CHANGELOG.md ?